src/auditlog/compat.py

```diff
@@ -6,10 +6,10 @@ def is_authenticated(user):
 
     Function provides compatibility following deprecation of method call to
     is_authenticated() in Django 2.0.
-
     """
-
-    if django.VERSION < (1, 10):
+    if not hasattr(user, 'is_authenticated'):
+        return False
+    if hasattr(user.is_authenticated, "__call__"):
```

Comment:
It would be better to use `callable()` for this check: https://bugs.python.org/issue10518#msg122584

Reply:
Ok, no problem. Am away for a few days but will either see if I can edit it through github or do it when I’m back. 

(You’re sure you want to check callable() instead of the version?)

Reply:
Didn't hear back, so changed as suggested.